version.props

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<!-- This file is needed for the VSO build in this location. -->
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MUXVersionMajor>2</MUXVersionMajor>
+    <MUXVersionMinor>2</MUXVersionMinor>
+    <MUXVersionBuild Condition="$(MUXVersionBuild) == ''">0</MUXVersionBuild>
+    <MUXVersionRevision Condition="$(MUXVersionRevision) == ''">0</MUXVersionRevision>
+  </PropertyGroup>
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <PreprocessorDefinitions>%(PreprocessorDefinitions);MUX_VERSION_MAJOR=$(MUXVersionMajor)</PreprocessorDefinitions>
```

Comment:
Why not just have one <PreprocessorDefinitions> tag?  It doesn't care about whitespace, so you could just stack them like this:

```
<PreprocessorDefinitions>
    %(PreprocessorDefinitions);
    MUX_VERSION_MAJOR=$(MUXVersionMajor);
    MUX_VERSION_MINOR=$(MUXVersionMinor);
    MUX_VERSION_BUILD=$(MUXVersionBuild);
    MUX_VERSION_REVISION=$(MUXVersionRevision);
</PreprocessorDefinitions>
```

Reply:
Nice. Much more readable.